src/qt/trafficgraphwidget.cpp

```diff
@@ -124,6 +124,7 @@ void TrafficGraphWidget::paintEvent(QPaintEvent *)
 
     const TrafficGraphData::SampleQueue& queue = trafficGraphData.getCurrentRangeQueueWithAverageBandwidth();
 
+    painter.setRenderHint(QPainter::Antialiasing);
```

Comment:
16153: this should apply to `drawPath`s only imo, see 1e996cb

Reply:
cherry-picked and squashed in latest push, thanks!